src/test/lib/logical_query_plan/lqp_translator_test.cpp

```diff
@@ -528,23 +526,53 @@ TEST_F(LQPTranslatorTest, PredicateNodePrunedIndexScan) {
   EXPECT_EQ(table_scan_op->excluded_chunk_ids, index_scan_chunk_ids);
   EXPECT_EQ(*table_scan_op->predicate(), *equals_(b, 42));
 
+  // When chunks have been pruned, the pruned ChunkIDs must be mapped to the original indexed ChunkIDs.
```

Comment:
See above: Do we really need this?

Reply:
Yes.
If I remove it --verify will fail.

Reply:
And what's the problem?

Reply:
Alright. Then at least do not create the mapping on an LQP level but in the operator so we do not have to change everything once we have dynamic chunk pruning.

Reply:
@tjjordan `std::vector<std::optional<ColumnID>> column_ids_after_pruning` might be relevant for this. With the pruned column ids, the mapping can be calculated within the operator and no additional mapping structure needs to be passed to the IndexScan's operator.

Reply:
I was referring to *chunk pruning*. Don't see an issue with pruned columns, which we will not touch in the operator for the sake of literally all expressions.